website/docs/d/scc_posture_scopes.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scopes"
+description: |-
+  Get information about list_scopes
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scopes
+
+Provides a read-only data source for list_scopes. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_scc_posture_scopes" "list_scopes" {
+	scope_id = "1"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `scope_id` - (Optional, String) An auto-generated unique identifier for the scope.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the list_scopes.
+* `scopes` - (Optional, List) Scopes.
+Nested scheme for **scopes**:
+	* `description` - (Optional, String) A detailed description of the scope.
+	* `created_by` - (Optional, String) The user who created the scope.
+	* `modified_by` - (Optional, String) The user who most recently modified the scope.
+	* `scope_id` - (Optional, String) An auto-generated unique identifier for the scope.
+	* `name` - (Optional, String) A unique name for your scope.
+	* `enabled` - (Optional, Boolean) Indicates whether scope is enabled/disabled.
+	* `environment_type` - (Optional, String) The environment that the scope is targeted to.
+	  * Constraints: Allowable values are: ibm, aws, azure, on_premise, hosted, services, openstack, gcp
+	* `created_time` - (Optional, String) The time that the scope was created in UTC.
+	* `modified_time` - (Optional, String) The time that the scope was last modified in UTC.
+	* `last_scan_type` - (Optional, String) The last type of scan that was run on the scope.
+	  * Constraints: Allowable values are: discovery, validation, fact_collection, fact_validation, inventory, remediation, abort_tasks, evidence, script
```

Comment:
Is constraints part of last_scan_type, if yes we need nested scheme....
Allowable values are: discovery, validation, fact_collection, fact_validation, inventory, remediation, abort_tasks, evidence,. as
Supported values are **discovery**..... in bold